package/kernel/mac80211/patches/ath12k/100-ath12k-prevent-ltssm-startup-crash.patch

```diff
@@ -0,0 +1,12 @@
+--- a/drivers/net/wireless/ath/ath12k/pci.c
++++ b/drivers/net/wireless/ath/ath12k/pci.c
+@@ -277,6 +277,9 @@ static void ath12k_pci_enable_ltssm(stru
+ 	u32 val;
+ 	int i;
+ 
++	/* Prevent startup crash on BPI-Rx */
++	return;
```

Comment:
Please report this problem, upstream so they can fix it.

Reply:
Hello,
Did it here in January.
https://lore.kernel.org/all/CAFED-jma7ZB17Rcgx9H+eVZ=N6Pty4__gz3B+O2V34XtcEzOGg@mail.gmail.com/T/

Didn't have time to dig more in that - find better workaround. So far this one works correctly - card also works correctly. From code I see they assume ioread32 from wrong addresss should report 0xffffffff - don't know this is true for all devices.

Reply:
Hello @hauke @januszdziedzic . Does it can cause this issue https://github.com/openwrt/openwrt/issues/17193 ?